.github/workflows/e2e.yaml

```diff
@@ -200,9 +199,8 @@ jobs:
       - name: Make ns executable
         run: chmod +x ns && mv ns /tmp/ns
 
-      - name: Login
```

Comment:
Let's keep the old login for `ns deploy --env=staging`

Reply:
True, reverted.